app/controllers/abouts_controller.rb

```diff
@@ -1,4 +1,5 @@
 class AboutsController < ApplicationController
+  skip_before_action :authenticate_user!
```

Comment:
```suggestion
  allow_unauthenticated_access
```
We have this helper method in Authentication module.